src/Symfony/Component/Console/Command/LockableTrait.php

```diff
@@ -18,7 +18,7 @@
 /**
  * Basic lock feature for commands.
  *
- * @author Fabien Potencier <fabien@symfony.com>
+ * @author Geoffrey Brier <geoffrey.brier@gmail.com>
```

Comment:
@geoffrey-brier Did I pick the right name and mail address (I took the data from your git commit)?